packages/kit/src/core/sync/write_client_manifest.js

```diff
@@ -104,4 +127,22 @@ export function write_client_manifest(config, manifest_data, output) {
 			};
 		`)
 	);
+
+	// write matchers to a separate module so that we don't
+	// need to worry about name conflicts
+	const imports = [];
```

Comment:
What's your reason for inlining this here?

Reply:
they're not really separate functions — the matchers are _part_ of the manifest, you can't (correctly) write one without the other. the only reason they're separate files at all is so we don't have to deconflict names